src/networks.json

```diff
@@ -1291,6 +1291,22 @@
     "start": 94085,
     "logo": "ipfs://QmaQxfwpXYTomUd24PMx5tKjosupXcm99z1jL1XLq9LWBS"
   },
+  "2152": {
+    "key": "2152",
+    "name": "Findora Mainnet",
+    "shortName": "Findora",
+    "chainId": 2152,
+    "network": "mainnet",
+    "multicall": "0x319E6CCaA89C3dCe65946E759a32a09531E6b72f",
```

Comment:
Thanks @ws4charlie It works great now, just that when I go here 
https://evm.findorascan.io/address/0x319E6CCaA89C3dCe65946E759a32a09531E6b72f

I can't verify the code of the contract

Reply:
@ChaituVR I fixed it by replacing old contract with a new verified one. Please check again:
https://evm.findorascan.io/address/0xCF7D1e21CBe9bdEF235aef06C5d8051B3d4DF0f5